testdata/p4tc_samples_outputs/add_entry_1_example.template

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -x
+
+set -e
+
+TC="tc"
+$TC p4template create pipeline/add_entry_1_example numtables 1
+
+$TC p4template create action/add_entry_1_example/MainControlImpl/send_nh actid 1 \
+	param dmac type bit48 \
+	param smac type bit48
+$TC p4template update action/add_entry_1_example/MainControlImpl/send_nh state active
+
+$TC p4template create action/add_entry_1_example/MainControlImpl/next_hop actid 2
+$TC p4template update action/add_entry_1_example/MainControlImpl/next_hop state active
+
+$TC p4template create action/add_entry_1_example/MainControlImpl/default_route_drop actid 3
+$TC p4template update action/add_entry_1_example/MainControlImpl/default_route_drop state active
+
+$TC p4template create table/add_entry_1_example/MainControlImpl/ipv4_tbl_1 \
+	tblid 1 \
+	type exact \
+	keysz 64 nummasks 8 tentries 2048 \
```

Comment:
Since this program is doing add on miss, the template script needs to give the table permissions for such.
So you should specify permissions to allow creating entries from the data path.
So you do something like:

```suggestion
	keysz 64 nummasks 8 tentries 2048 permissions 0x3DE6 \
```

Reply:
added.